### ISSUE TITLE
.NET Core Support

### ISSUE THREAD

**Issue by mysteryx93** [lbugnion/mvvmlight#88]
When I add MvvmLight to a .NET Core project, I get this error

> Warning	NU1701	Package 'MvvmLightLibs 5.4.1.1' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETCoreApp,Version=v3.1'. This package may not be fully compatible with your project.

Has it been updated to work with .NET Core?

**Comment by jamesmcroft** [comment]
I believe you get .NET core support from the MvvmLightLibsStd10 (.NET Standard) package, however, it hasn't been updated since 2018 so might not give you the functionality you might like for WPF.

**Comment by mysteryx93** [comment]
Ah I see. Only problem is that GalaSoft.MvvmLight.CommandWpf won't be supported.

Or keep it with the .NET Framework version; it's working but it's just showing the warnings.

**Comment by LentoMan** [comment]
Until we get any official statement from Laurent, you could always try to (look for a) fork.

Another option is to try a similar library like Prism, which currently supports Wpf for .NETCoreApps

**Comment by KevinBassaDevelopment** [comment]
.Net Core support would be great!

**Comment by jamesmcroft** [comment]
I don't think there has been any official statement on whether MvvmLight is no longer going to be maintained. 

There has been a bit of effort to breathe new life into MvvmLight through building a new MVVM framework within the [Windows Community Toolkit](https://github.com/windows-toolkit/WindowsCommunityToolkit/tree/master/Microsoft.Toolkit.Mvvm). 

The umbrella for this says Windows but don't worry. The MVVM Toolkit there which was inspired by the MvvmLight toolkit is platform agnostic.

**Comment by mysteryx93** [comment]
Sounds like an awesome initiative! I've looked at the other MVVM frameworks and I didn't find the extra features really useful for my specific needs. I prefer it does a basic job well.

A well-supported rewrite with better performance would be a double-plus!

**Comment by michael-hawker** [comment]
Figured I'd follow-up here and say that we shipped our 7.0 version last month which includes the new MVVM Toolkit. You can find the notes on migrating from MVVM Light here: https://docs.microsoft.com/en-us/windows/communitytoolkit/mvvm/migratingfrommvvmlight and samples here: https://aka.ms/mvvmtoolkit/samples.

**Comment by luetm** [comment]
> Figured I'd follow-up here and say that we shipped our 7.0 version last month which includes the new MVVM Toolkit. You can find the notes on migrating from MVVM Light here: https://docs.microsoft.com/en-us/windows/communitytoolkit/mvvm/migratingfrommvvmlight and samples here: https://aka.ms/mvvmtoolkit/samples.

Does that work with WPF too?

**Comment by michael-hawker** [comment]
@luetm yes, the MVVM Toolkit is based on .NET Standard and works with WPF, UWP, Xamarin, Uno, MAUI, WinUI 3, etc...
